### PR TITLE
[FIX] web_responsive: Can't access web using superuser

### PR DESCRIPTION
**Pull request by CarlosRoca13**
Before this change when trying to access web using superuser the web interface is not charged.

By doing this change is all working normally.

Steps to reproduce the problem:

1. Log in
2. Go to web/become

cc @Tecnativa 

ping @pedrobaeza @chienandalu @mohamedhagag

This PR tries to solve this issue: https://github.com/OCA/web/issues/2904

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please attend pre-commit.

**Review by pedrobaeza — APPROVED** [review]
Tested and working

**Comment by CarlosRoca13** [comment]
Done! :smile:

File: web_responsive/static/src/components/apps_menu/apps_menu.esm.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ patch(WebClient.prototype, {
                 ["is_redirect_home"]
             );
             this.env.services.user.updateContext({
-                is_redirect_to_home: is_redirect_home[0].is_redirect_home,
+                is_redirect_to_home:
+                    is_redirect_home[0] && is_redirect_home[0].is_redirect_home,
```
Comment (chienandalu) [inline web_responsive/static/src/components/apps_menu/apps_menu.esm.js]: In v17 you should be able to use js [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) :smile: !
```suggestion
                is_redirect_to_home: is_redirect_home?.[0]?.is_redirect_home,
```

Comment (CarlosRoca13) [reply web_responsive/static/src/components/apps_menu/apps_menu.esm.js]: Oh! I didn't know about it :sweat_smile: Thanks!

**Comment by CarlosRoca13** [comment]
pre-commit is not happy with this change :cry:

**Comment by chienandalu** [comment]
check: https://github.com/OCA/social/pull/1378

**Comment by CarlosRoca13** [comment]
Added the update :smile:

**Review by chienandalu — APPROVED** [review]

**Pull request merged**